soc/arm/nuvoton_npcx/common/scfg.c

```diff
@@ -4,7 +4,7 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-#define DT_DRV_COMPAT nuvoton_npcx_pinctrl
+#define DT_DRV_COMPAT nuvoton_npcx_scfg
 
 #include <drivers/pinmux.h>
```

Comment:
The include should be removed.

Reply:
It looks like, the include hasn't been removed yet.

Reply:
Done.